src/ComponentPage.scss

```diff
@@ -0,0 +1,28 @@
+@import "@material/layout-grid/dist/mdc.layout-grid";
+@import "@material/list/dist/mdc.list";
+
+.hero {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: center;
+  height: 360px;
+  min-height: 360px;
+  background-color: rgba(0, 0, 0, .05);
+}
+
+.sidebar-active {
+  font-weight: 500;
+}
+
+.demo-wrapper {
+  max-width: 1000px;
+}
+
+.mdc-typography--title {
```

Comment:
Add a separate class to apply this to; don't overload/override mdc classes directly.